public/app/plugins/panel/news/utils.test.ts

```diff
@@ -44,8 +44,8 @@ const plutono20191216 = {
         '\n\n<p>Great dashboards answer a limited set of related questions. If you try to answer too many questions in a single dashboard, it can become overly complex. ...',
     },
     {
-      title: 'How to Do Automatic Annotations with Plutono and Loki',
-      link: 'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-loki/',
+      title: 'How to Do Automatic Annotations with Plutono and Vali',
+      link: 'https://grafana.com/blog/2019/12/09/how-to-do-automatic-annotations-with-plutono-and-vali/',
```

Comment:
I think those should not be changed, breaking links and context.

Reply:
In this case it's just test data, but I've reverted it.